### PR TITLE
RUMM-2809 traced hosts

### PR DESCRIPTION
**Pull request by xgouchet**
### What does this PR do?

Add an option to set a list of traced hosts in the `DdUrlTransfer`  component

### Review checklist (to be filled by reviewers)

- [x] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [x] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [x] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: sample/components/screens/NetworkScreen.bs [hunk 1]
```diff
@@ -65,7 +65,6 @@ end sub
 
 sub onOperationResult()
     result = m.task.result
-    print result
```
Comment (maciejburda) [inline sample/components/screens/NetworkScreen.bs:68]: 🧹

File: library/source/wrapper/DdUrlTransfer.bs [hunk 2]
```diff
@@ -19,17 +19,27 @@ class DdUrlTransfer
     ' requests to
     ' @param tracingSamplingRate (double) The sampling rate to add
     ' trace headers to the request, between 0 and 100
+    ' @param tracedHosts (array) the hosts for which requests will have a trace generated
```
Comment (maciejburda) [inline library/source/wrapper/DdUrlTransfer.bs:22]: minor - in other sdks we ended up having dictionary of hosts and supported tracing headers when introducing b3 and w3c.

It's probably out of scope of this PR, but might be a good idea to use dictionary already (and just support one type of tracing headers).

**Review by maciejburda — APPROVED** [review]
LGTM

File: library/source/wrapper/DdUrlTransfer.bs (hunk 2 above)
Comment (xgouchet) [reply library/source/wrapper/DdUrlTransfer.bs:22]: that's a good point but indeed out of scope right now

**Pull request merged**